rush-plugins/rush-amazon-s3-build-cache-plugin/src/test/AmazonS3Credentials.test.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  AWS_ACCESS_KEY_ID,
+  AWS_SECRET_ACCESS_KEY,
+  AWS_SESSION_TOKEN,
+  fromAmazonEnv,
+  fromRushEnv
+} from '../AmazonS3Credentials';
+import { EnvironmentConfiguration } from '@rushstack/rush-sdk';
+
+describe('Amazon S3 Credentials', () => {
+  describe(fromAmazonEnv.name, () => {
+    afterEach(() => {
+      delete process.env[AWS_ACCESS_KEY_ID];
+      delete process.env[AWS_SECRET_ACCESS_KEY];
+      delete process.env[AWS_SESSION_TOKEN];
+    });
```

Comment:
Can you back up any existing values these might have first and restore them afterward so `process.env` can't be permanently modified by this test?